packages/lib-react-components/src/Markdownz/Markdownz.spec.js

```diff
@@ -11,4 +13,222 @@ describe('<Markdownz />', function () {
+    })
+
+    it('should return true if the symbol is @ and the resource is not in the props.restrictedUserNames array', function () {
+      wrapper.instance().shouldResourceBeLinkable('srallen', '@')
+      expect(shouldResourceBeLinkableSpy).to.have.returned(true)
+    })
+
+    it('should return false if the symbol is @ and the resource is in the props.restrictedUserNames array', function () {
+      wrapper.instance().shouldResourceBeLinkable('team', '@')
+      expect(shouldResourceBeLinkableSpy).to.have.returned(false)
+    })
+  })
+
+  describe('#renderMedia', function () {
+    const videoPropsMock = { src: 'https://static.zooniverse.org/www.zooniverse.org/assets/home-video.mp4', alt: 'Video of the Zooniverse', children: undefined }
+    const audioPropsMock = { src: 'https://panoptes-uploads.zooniverse.org/production/subject_location/1c93591f-5d7e-4129-a6da-a65419b88048.mpga', alt: 'Street noise', children: undefined }
+    let renderMediaSpy
+    before(function () {
+      renderMediaSpy = sinon.spy(Markdownz.prototype, 'renderMedia')
+    })
+    afterEach(function () {
+      renderMediaSpy.resetHistory()
+    })
+    after(function () {
+      renderMediaSpy.restore()
+    })
+
+    describe('when the media is an image', function () {
+      const imagePropsMock = { src: 'https://via.placeholder.com/350x350', alt: 'image-alt-text', children: undefined }
+      const imagePropsMockWithSize = { src: 'https://via.placeholder.com/350x350', alt: 'image-alt-text =100x100', children: undefined }
+      it('should return a Grommet Image component', function () {
+        wrapper.instance().renderMedia(imagePropsMock)
+        const returnedValue = renderMediaSpy.returnValues[0]
+        expect(returnedValue.type).to.equal(Image)
+      })
+
+      it('should set the alt attribute with the alt prop', function () {
+        wrapper.instance().renderMedia(imagePropsMock)
+        const returnedValue = renderMediaSpy.returnValues[0]
+        expect(returnedValue.props.alt).to.equal(imagePropsMock.alt)
+      })
+
+      it('should set the src attribute with the src prop', function () {
+        wrapper.instance().renderMedia(imagePropsMock)
+        const returnedValue = renderMediaSpy.returnValues[0]
+        expect(returnedValue.props.src).to.equal(imagePropsMock.src)
+      })
+
+      it('should use the width and height from the alt text if defined', function () {
+        wrapper.instance().renderMedia(imagePropsMockWithSize)
+        const returnedValue = renderMediaSpy.returnValues[0]
+        expect(returnedValue.props.width).to.equal('100')
+        expect(returnedValue.props.height).to.equal('100')
+      })
+
+      it('should remove the width and height declaration from the alt text before setting it on the rendered Image', function () {
+        wrapper.instance().renderMedia(imagePropsMockWithSize)
+        const returnedValue = renderMediaSpy.returnValues[0]
+        expect(returnedValue.props.alt).to.equal('image-alt-text')
```

Comment:
Expected alt text here should be changed too. Maybe store the alt text string in a constant rather than repeating it throughout the test function.